tests/test_utils.py

```diff
@@ -32,5 +32,6 @@ def test_parse_redirected_url(self):
         r = parse_redirected_url(redirected, flow)
         self.assertEqual(r, code)
 
-    def tearDown():
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
Since TestUtils, is the last test to run, this method will close the session and we no longer will get the unclosed-socket warnings. But there's still a problem.
I ran the tests for 2.0.0 and in the end, only got one warning. But at the end of the tests for the current version, there are two warnings. This might mean that we are opening two sockets which is unnecessary. At the same time, `genius._session.close()` makes both of the warnings go away which is curious.